### PR TITLE
Closed tests

### PR DESCRIPTION
**Pull request by wpride**

### CODE REVIEW

File: touchforms/backend/test_touchcare.py [hunk 1]
```diff
@@ -144,5 +144,50 @@ def setUp(self):
 
         self.filter_expr = "[case_name = 'Napoli']"
 
+class ParentClosedTests(unittest.TestCase):
```
Comment (dimagimon) [inline touchforms/backend/test_touchcare.py]: E302 expected 2 blank lines, found 1

File: touchforms/backend/test_touchcare.py [hunk 2]
```diff
@@ -144,5 +144,50 @@ def setUp(self):
...
+            self.session_data,
+            restore_xml=self.restore,
+            uses_sqlite=True,
+        )
+
+        self.assertEqual(len(resp['cases']), 2)
+
 if __name__ == '__main__':
     unittest.main()
+
```
Comment (dimagimon) [inline touchforms/backend/test_touchcare.py]: W293 blank line contains whitespace

Comment (dimagimon) [inline touchforms/backend/test_touchcare.py]: W391 blank line at end of file

**Comment by benrudolph** [comment]
dimagimon :yum: (note you can always add `[ci skip]` to the commit so the tests don't run again)

**Pull request merged**